lib/App/index.js

```diff
@@ -711,7 +668,26 @@ class App {
       });
 
 		})
-	}
+  }
+  
+  async _getPackFileEntries(prunePaths) {
+    const walker = new ignoreWalk.Walker({
+      path: this.path,
+      ignoreFiles: [ '.homeyignore', '' ],
+      includeEmpty: true,
+      follow: true
+    });
+
+    // Add default ignore rules for dotfiles, env.json and npm prune paths
+    const ignoreRules = ['.*', 'env.json', ...prunePaths];
+    walker.onReadIgnoreFile('', ignoreRules.join('\r\n'), () => {});
```

Comment:
Also, here, could you clarify this perhaps?

Reply:
I think I covered it in the previous discussion. No, I'm not so happy with it, but manually filtering the dev dependencies given only partial paths was a much larger pain in the butt.